css assignment/gallery.html

```diff
@@ -16,54 +16,50 @@
         <a href="./gallery.html" class="items"> My Gallery </a>
         
     </nav>
+    <h1 class="heading" style="text-align:center;">Gallery</h1>
 
-    <div class="container">
-      <h1 class="heading">Gallery</h1>
-      <ul>
-        <li>
+    <div class="flex-container">
```

Comment:
can you make all gallary img content to the center

now it is more on left side

Reply:
Done.